posthog/api/decide.py

```diff
@@ -291,13 +291,23 @@ def get_decide(request: HttpRequest):
     return cors_response(request, JsonResponse(response))
 
 
-def _session_recording_config_response(request: HttpRequest, team: Team) -> bool | dict:
+def _session_recording_config_response(request: HttpRequest, team: Team, token: str) -> bool | dict:
     session_recording_config_response: bool | dict = False
 
     try:
         if team.session_recording_opt_in and (
             on_permitted_recording_domain(team, request) or not team.recording_domains
         ):
+            if settings.EE_AVAILABLE:
```

Comment:
I like the idea but can we add another flag for this. We haven't tested that function in decide and any number of things could go wrong so would nice to be able to revert easily here.

Reply:
Also a test would be really necessary here

Reply:
yeah i wanted feedback on whether it was a complete non-starter for some reason i hadn't thought of before adding tests

Reply:
Instead of depending on a revert, I would have suggested a feature flag here. But that would end up creating an ouroboros. The irony of using feature flags inside the feature flag endpoint :)